Yoast/wordpress-seo
# Always show the config wizard banner

## Summary

This PR can be summarized in the following changelog entry:

* We now _always_ show the configuration wizard banner on Dashboard → Dashboard. 
* The Credits link has been moved from Dashboard → General to the bottom of Dashboard → Dashboard.
* The Credits page within the plugin has been replaced by a Credits page on Yoast.com.
* As Dashboard → General is now empty, I've removed it.

## Test instructions

This PR can be tested by following these steps:

* Open the Dashboard, see the configuration wizard notice. 
* See the credits link.

## Quality assurance


Fixes #8977
